apps/installer-stub/src/styles.css

```
@@ -0,0 +1,59 @@
+:root {
+  color-scheme: dark;
+  background: #101214;
+  color: #f4f6f8;
+  font-family: ui-monospace, "SFMono-Regular", "SF Mono", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.10.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)